product_variant_configurator/views/inherited_product_template_views.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+
+    <record model="ir.ui.view" id="product_template_no_variant_form_view">
+        <field name="name">product.template.no.variant.form</field>
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="product.product_template_only_form_view" />
+        <field name="arch" type="xml">
+            <div name="button_box" position="inside">
+                <button class="oe_stat_button"
+                        name="action_open_attribute_prices"
+                        attrs="{'invisible': [('attribute_line_ids', '=', [])]}"
+                        groups="product.group_product_variant"
+                        icon="fa-strikethrough"
+                        type="object"
+                        string="Variant Sale Prices"/>
```

Comment:
Variant Sale Prices stat buttun is visible when there is not variables created yet. This is the expected behavior?

Reply:
Yes, It's showed if you have atributes, I think it's not needed to check if variants are created.